src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```
@@ -0,0 +1,29 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).get();
```

let's throw an exception with an informative message in case optional has null inside